### PR TITLE
docs: replace the reference to co on the async function

### PR DESCRIPTION
**Pull request by vkhv**
Replaced the reference to co on the async function

### CODE REVIEW

**Comment by fl0w** [comment]
Well, technically, koa@2 is a Promise based library. Not trying to be techo-fascist or a purist here, but I subjectively disagree with the statement "uses async functions".

`async-await` is just sugar for yielding anyway, IMO. I'm -1.

File: docs/koa-vs-express.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ THIS DOCUMENT IS IN PROGRESS. THIS PARAGRAPH SHALL BE REMOVED WHEN THIS DOCUMENT
 # Koa vs Express
 
   Philosophically, Koa aims to "fix and replace node", whereas Express "augments node".
-  Koa uses co to rid apps of callback hell and simplify error handling.
+  Koa uses async functions to rid apps of callback hell and simplify error handling.
   It exposes its own `this.request` and `this.response` objects instead of node's `req` and `res` objects.
```
Comment (wong2) [inline docs/koa-vs-express.md]: this could also be updated

Comment (vkhv) [inline docs/koa-vs-express.md]: 6797d348399ac7d62dde10e5f72faae6df5189bf

File: docs/koa-vs-express.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ THIS DOCUMENT IS IN PROGRESS. THIS PARAGRAPH SHALL BE REMOVED WHEN THIS DOCUMENT
 # Koa vs Express
 
   Philosophically, Koa aims to "fix and replace node", whereas Express "augments node".
-  Koa uses co to rid apps of callback hell and simplify error handling.
-  It exposes its own `this.request` and `this.response` objects instead of node's `req` and `res` objects.
+  Koa uses async functions to rid apps of callback hell and simplify error handling.
+  It exposes its own `cxt.req` and `ctx.res` objects instead of node's `req` and `res` objects.
```
Comment (wong2) [inline docs/koa-vs-express.md]: should be `ctx.request` and `ctx.response`

Comment (vkhv) [inline docs/koa-vs-express.md]: sry, 6797d348399ac7d62dde10e5f72faae6df5189bf

Comment (vkhv) [inline docs/koa-vs-express.md]: sry, c08ddb597edbc4b1ed79edc11ff2709565aa4a01

**Comment by jonathanong** [comment]
merged https://github.com/koajs/koa/commit/df7b4ff2b882914748d37ae17de951e62d4b3bd4 with edits

squashed badly and removed your authorship. sorry :(

**Pull request closed without merging**